docs/source/install/__mongodb_note.rst

```diff
@@ -1,9 +1,8 @@
 .. note::
 
-  The currently supported versions of MongoDB are 3.4 and 4.0. This is the version installed by
-  the installer script. MongoDB 4.0 is installed by default on Ubuntu 18.04 and RHEL/CentOS8.
+  The currently supported version of MongoDB is 4.0. This is the version installed by
+  the installer script. 
 
-  MongoDB 3.6 is also supported by StackStorm >= 3.0.0, but we have observed some
-  performance regressions with MongoDB 3.6 so the default version which is installed on Ubuntu
-  Xenial (16.04) and EL7 (CentOS 7 and RHEL 7) is still 3.4.
-  
\ No newline at end of file
+  MongoDB 3.4 and 3.6 are also supported by StackStorm >= 3.0.0, but we have observed some
+  performance regressions with MongoDB 3.6.
```

Comment:
```suggestion
```
This could be removed now as we do care about MongoDB 4.0 from now since https://github.com/StackStorm/st2/issues/4972.

Also as said in https://github.com/StackStorm/st2/issues/4972#issuecomment-659496335 if possible, it's always better to avoid mixing several logically independent changes in one PR like EL6 deprecation, MongoDB 4.0, EL8 link update or something else.